[2024-04-25T06:20:59.863+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-25T06:20:59.876+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_minio_s3_v02.sensor_minio_s3 scheduled__2022-10-01T00:00:00+00:00 [queued]>
[2024-04-25T06:20:59.897+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_minio_s3_v02.sensor_minio_s3 scheduled__2022-10-01T00:00:00+00:00 [queued]>
[2024-04-25T06:20:59.905+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-25T06:20:59.920+0000] {taskinstance.py:2327} INFO - Executing <Task(S3KeySensor): sensor_minio_s3> on 2022-10-01 00:00:00+00:00
[2024-04-25T06:20:59.932+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=12181) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-04-25T06:20:59.933+0000] {standard_task_runner.py:63} INFO - Started process 12189 to run task
[2024-04-25T06:20:59.929+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_with_minio_s3_v02', 'sensor_minio_s3', 'scheduled__2022-10-01T00:00:00+00:00', '--job-id', '2109', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_minio_s3.py', '--cfg-path', '/tmp/tmpf7exyle5']
[2024-04-25T06:20:59.938+0000] {standard_task_runner.py:91} INFO - Job 2109: Subtask sensor_minio_s3
[2024-04-25T06:21:00.013+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_with_minio_s3_v02.sensor_minio_s3 scheduled__2022-10-01T00:00:00+00:00 [running]> on host a55babecb56c
[2024-04-25T06:21:00.087+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_minio_s3_v02' AIRFLOW_CTX_TASK_ID='sensor_minio_s3' AIRFLOW_CTX_EXECUTION_DATE='2022-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-10-01T00:00:00+00:00'
[2024-04-25T06:21:00.088+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-25T06:21:00.104+0000] {baseoperator.py:399} WARNING - S3KeySensor.execute cannot be called outside TaskInstance!
[2024-04-25T06:21:00.107+0000] {s3.py:110} INFO - Poking for key : s3://***/data.csv
[2024-04-25T06:21:00.116+0000] {base.py:84} INFO - Using connection ID 'minio_conn' for task execution.
[2024-04-25T06:21:00.117+0000] {connection_wrapper.py:387} INFO - AWS Connection (conn_id='minio_conn', conn_type='aws') credentials retrieved from extra.
[2024-04-25T06:21:00.119+0000] {logging_mixin.py:188} WARNING - <string>:9 AirflowProviderDeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead
[2024-04-25T06:21:00.285+0000] {base.py:294} INFO - Success criteria met. Exiting.
[2024-04-25T06:21:00.286+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-25T06:21:00.296+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_with_minio_s3_v02, task_id=sensor_minio_s3, execution_date=20221001T000000, start_date=20240425T062059, end_date=20240425T062100
[2024-04-25T06:21:00.324+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-25T06:21:00.336+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-25T06:21:00.338+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
